tests/integration/utils/mock-api.js

```diff
@@ -44,6 +44,12 @@ export const startMockApi = ({ routes, silent }) => {
     res.json({ site_id: 'site_id', integrations: [] })
   })
 
+  app.get('/team/team_id/integrations/installations/meta/site_id', function onRequest(req, res) {
```

Comment:
I'm going to just trust you on this one 🤷🏼

Reply:
I mean same.